config/fusor-installer.answers.yaml

```diff
@@ -38,7 +38,12 @@ capsule:
 
 foreman::plugin::bootdisk: true
 foreman::plugin::discovery:
+  # We do not want to seek the images from the internet.
+  # want them installed from RPM, foreman-discovery-image
   install_images: false
+  # Overriding kernel/initrd to match what is in foreman-discovery-image (needed as of 2/27/15)
```

Comment:
@jwmatthews, Ok, I didn't know if the "needed as of" was implying that there is an issue in the current discovery which requires us to specify these values.